app/src/main/java/de/Maxr1998/xposed/maxlock/ui/LockActivity.java

```diff
@@ -38,35 +39,61 @@
 public class LockActivity extends FragmentActivity implements AuthenticationSucceededListener {
 
     private String requestPkg;
-    private ActivityManager am;
     private Intent app;
-    private SharedPreferences prefsPackages;
+    private SharedPreferences prefsPackages, prefs;
     private boolean isInFocus = false, unlocked = false;
 
     @SuppressLint("WorldReadableFiles")
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         Util.cleanUp(this);
-        // Preferences
+        //Preferences
         //noinspection deprecation
         prefsPackages = getSharedPreferences(Common.PREFS_PACKAGES, MODE_WORLD_READABLE);
+        prefs = PreferenceManager.getDefaultSharedPreferences(this);
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_lock);
 
         requestPkg = getIntent().getStringExtra(Common.INTENT_EXTRAS_PKG_NAME);
         app = getIntent().getParcelableExtra(Common.INTENT_EXTRAS_INTENT);
 
-        am = (ActivityManager) getSystemService(Activity.ACTIVITY_SERVICE);
-        am.killBackgroundProcesses("de.Maxr1998.xposed.maxlock");
+        //////////am = (ActivityManager) getSystemService(Activity.ACTIVITY_SERVICEX);
+        //////////am.killBackgroundProcesses("de.Maxr1998.xposed.maxlock"X);
+        //FUUUUUCCCCCCCCCKKKKKKKKKK WHO IS THE DUMB THAT PUTTED A KILL HERE LOST 2 DAYS BECAUSE OF IT !
+        //I know it's not maxr but if you really care about background app then hook the oncreate activity instead of calling 2 new activity.class
```

Comment:
Well, I also wondered why to use this, this was still from @chiehmin.. I removed it everywhere in my code now.